docs/developer/community/faq.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Frequently Asked Questions
+sidebar_label: FAQ
+---
+
+## I cannot log in to the dashboard. I receive an error: ‘saleor ran into an unexpected problem’. What actions should I take?
+
+Make sure that you have set the environment variables.
+
+There are two environment variables available for configuration:
+
+- `API_URI` (required): URI of the GraphQL API endpoint of a running instance of Saleor. If you are running Saleor locally with the default settings, set `API_URI` to: `http://localhost:8000/graphql/`.
+
+- `APP_MOUNT_URI`: URI at which the Dashboard app will be mounted. E.g. if you set `APP_MOUNT_URI` to `/dashboard/``, your app will be mounted at `http://localhost:9000/dashboard/`.
+
+- `STATIC_URL`: URL where the static files are located. E.g. if you use S3 bucket, you should set it to the bucket's URL. By default, Saleor assumes you serve static files from the root of your site at `http://localhost:9000/`.
+
+You can find the instructions on how to install and run the dashboard here:
+
+https://github.com/mirumee/saleor-dashboard#getting-started
+
+If you still cannot log in to the dashboard, please contact us on Spectrum.
+
+## How to make Saleor work in production with Ubuntu 19.10/Nginx/Gunicorn server?
+
+You will need to perform the following actions:
+
+- Build the storefront and the dashboard.
+
+- Upload both to separate folders on the server.
+
+- Create separate location entries in your Nginx config for the backend, the dashboard, and the storefront.
+
+    ```
+    location /graphql/ {
+        proxy_pass http://localhost:8000;
+    }
+
+    location / {
+        root /path/to/your/storefront;
+    }
+
+    location /dashboard/ {
+        root /path/to/your/dashboard;
+    }
+    ```
+
+If you still have any issues, please contact us on Spectrum.
+
+## I receive an error 405. What does it mean?
+
+- Please check if you have set the environment variables API_URI
+
+- Please check if the app is actually connected to the API
+
+- You can investigate the connection in the Network tab in your browser’s dev tools. Please look for GraphQL requests.
+
+If you still have any issues, please [contact us on Spectrum](https://spectrum.chat/saleor) or [Gitter](https://gitter.im/mirumee/saleor).
+
+## How can I run Saleor locally?
+
+We’ve recently introduced a new repository: [saleor-platform](https://github.com/mirumee/saleor-platform). You can run all Saleor components (GraphQL API, Storefront, Dashboard) from one place by using Docker. Please, bear in mind that Saleor-Platform repository has only been designed for local development. If you still have any issues, please contact us on Gitter or Spectrum.
```

Comment:
"please contact us on Gitter or Spectrum." - maybe instead of saying it over and over in each answer, we should say at the top or at the bottom something like "If you didn't find a solution to your problem, contact us on Gitter (link) or Spectrum (link)"?

Reply:
Good point. Maybe we can write something like this: 
"Haven't found the answer you were looking for? Contact us on Gitter or Spectrum"